### PR TITLE
Tweaked usage message

### PR DESCRIPTION
**Pull request by DennisLfromGA**
Tweaked USAGE to be consistent with startxiwi.

### CODE REVIEW

File: chroot-bin/xiwi [hunk 1]
```diff
@@ -7,17 +7,21 @@
 
 USAGE="Usage: ${0##*/} [-f] [-F|-T] APPLICATION [PARAMETERS ...]
 Launches a windowed session in Chromium OS for any graphical application.
+Applications launched in this way show in independent windows or tabs.
 All parameters are passed to the specified application.
 
-By default, the app is launched in a window. Passing -F will launch the app
```
Comment (dnschneid) [inline chroot-bin/xiwi]: I think you lost the "by default" part.

**Comment by DennisLfromGA** [comment]
I lost it intentionally because I added the Options: section.
Maybe this file should be left alone?

**Comment by dnschneid** [comment]
I don't see any mention of what the default behavior is, though.  Your startxiwi help text does describe it.

File: chroot-bin/xiwi (hunk 1 above)
Comment (DennisLfromGA) [inline chroot-bin/xiwi]: Your right, sorry, I'll put it back in.
I guess I glossed over the `By default, the app is launched in a window.` part.

**Pull request merged**